docker/vendor/fluentd/clean-apt

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+# Copyright 2016 Astronomer Inc.
```

Comment:
Given this was taken from kube protect we should maintain their copy right I think - especially when we don't change the file.

Or at least add an attribution/source in a comment

Reply:
Probably keep the original copyright line and add ours below